README.md

```diff
@@ -179,9 +179,8 @@ For serial execution use:
 ```
 > lein modules utest
 ```
-Alternativly unit tests can be run in parallel using the python script in
-[run_unit_tests.py][ut-script]. This script is meant to be called with by a
-build system such as Bamboo.
+Alternatively unit tests can be run in parallel using the python script in
+[run_unit_tests.py][ut-script]. This script is meant to be called with by with a build system such as Bamboo.
```

Comment:
This line doesn't read correctly.  replace 
"called with by with a build system" 
with 
called by a build system